x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/hooks/use_create_analytics_form/use_create_analytics_form.ts

```diff
@@ -125,25 +125,63 @@ export const useCreateAnalyticsForm = (): CreateAnalyticsFormProps => {
 
   const createKibanaIndexPattern = async () => {
     const dataViewName = destinationIndex;
+    let retryCount = 5;
 
     try {
-      await mlContext.dataViewsContract.createAndSave(
-        {
-          title: dataViewName,
-        },
-        false,
-        true
-      );
+      const interval = setInterval(async () => {
+        // Ran out of retry attempts
+        if (retryCount === 0) {
+          clearInterval(interval);
+          addRequestMessage({
+            error: 'No destination index check attempts left.',
+            message: i18n.translate(
+              'xpack.ml.dataframe.analytics.create.unableToCreateDataViewForDataFrameAnalyticsJob',
+              {
+                defaultMessage: 'Unable to create data view for:',
```

Comment:
The displayed error message looks incomplete:
![image](https://user-images.githubusercontent.com/1945390/144458226-258e5e1a-4555-41a8-85e8-000e646ff4f6.png)

Maybe just `Unable to create data view` or add the destination index name?

Reply:
Updated to `Unable to create data view.` for the tile and `Destination index has not yet been created.` for the body in e270e5e074d38a9a6c9861c45e2bebb836efbc9d